api_tests/preprints/views/test_preprint_institutions_relationship.py

```diff
@@ -17,23 +13,37 @@ def user(self):
         return AuthUserFactory()
 
     @pytest.fixture()
-    def admin_with_institutional_affilation(self, institution, preprint):
+    def admin_with_institutional_affiliation(self, institution, preprint):
+        user = AuthUserFactory()
+        preprint.add_contributor(user, permissions=ADMIN)
+        user.add_or_update_affiliated_institution(institution)
+        return user
```

Comment:
For all the fixtures, if not too much effort, can we add more than one affiliation to the user to test more variations. e.g.

* Admin has A, B, C
* Write has B, C, D
* Read has C, E, F